kmcd-props.md

```diff
@@ -137,65 +138,150 @@ Violations
 A violation occurs if any of the properties above holds.
 
 ```k
-    syntax Bool ::= violated(List) [function, functional]
- // -----------------------------------------------------
-    rule violated(EVENTS) => zeroTimePotInterest(EVENTS)
-                      orBool unAuthFlipKick(EVENTS)
-                      orBool unAuthFlapKick(EVENTS)
-                      orBool potEndInterest(EVENTS)
+    syntax Map ::= "#violationFSMs" [function]
+ // ------------------------------------------
+    rule #violationFSMs => ( "Zero-Time Pot Interest Accumulation" |-> zeroTimePotInterest )
+                           ( "Pot Interest Accumulation After End" |-> potEndInterest      )
+                           ( "Unauthorized Flip Kick"              |-> unAuthFlipKick      )
+                           ( "Unauthorized Flap Kick"              |-> unAuthFlapKick      )
+                           ( "Total Bound on Debt"                 |-> totalDebtBounded(1) )
```

Comment:
Why `1` here? Naively, I'd expect zero since the system starts off with no debt.

Reply:
The `1` here is actually DSR, not `Debt`. So we are assuming we start at 0% interest on DSR.

Reply:
Ah, that's right, thanks.